docs/source/notebooks/multilevel_modeling.ipynb

```diff
@@ -69,8 +69,8 @@
     "import numpy as np\n",
```

Comment:
<p>Isn't <code>warning</code> a built-in library? If such, I <em>think</em> it should be the very first import, in a first block</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
@AlexAndorra yes - there is no block between them because of `no_sections=1`, which I put because the style guide ony has

>    Absolute imports first (import xyz as x)
>    Relative imports second (from xyz import function)

So, would you like to have standard library imports separated from third-party ones?

Reply:
Ooh you're right, the style guide isn't clear enough here. I think separating standard library imports from third-party ones is pep8-compliant, so it would probably better to do that. Do you have any preference?

Reply:
Agreed - that's actually the `isort` default, so doing it this way means less configuration :)